Bestpractices.md

```diff
@@ -8,15 +8,45 @@ Whether you are building an App for your own company or for all of Zendesk custo
 
 If you aren't sure about any of the points below, if you have questions about the approval process or if you would like to receive guidance or help building your first Zendesk App, please feel free to get in touch at support@zendesk.com.
 
-## Zendesk Apps: Must avoid
+## General Advice
 
-:no_entry: In this section you'll find practices you must avoid if you want your App to pass the approval process :no_entry:
+* DRY. Be critical of your own app, if you see that you've repeatedly done the same thing in your code then look to simplify by making helper functions that can be used in more than one place.
```

Comment:
Should we make it obvious this is an acroynym? ie. `D.R.Y.` or even just spell it out - "Don't Repeat Yourself"